src/tz.cpp

```diff
@@ -3627,7 +3712,17 @@ locate_zone(std::string_view tz_name)
 locate_zone(const std::string& tz_name)
 #endif
 {
-    return get_tzdb().locate_zone(tz_name);
+    return get_tzdb().locate_zone(
+#if WIN32 && USE_OS_TZDB
+    // When USE_OS_TZDB=ON, the tz_name needs to be a valid path.
+    // For timezone labeling consistency in Windows, switch "/" to "\".
+    // This will allow for tz_name to have consistent naming regardless of
+    // if USE_OS_TZDB is ON.
+    std::regex_replace(tz_name, std::regex("\\/"), "\\")
+#else
+    tz_name
+#endif
+    );
```

Comment:
Instead of using `std::regex`, why not `std::replace`? Here you just need to replace a single character.

```suggestion
#if WIN32 && USE_OS_TZDB
    // When USE_OS_TZDB=ON, the tz_name needs to be a valid path.
    // For timezone labeling consistency in Windows, switch "/" to "\".
    // This will allow for tz_name to have consistent naming regardless of
    // if USE_OS_TZDB is ON.
    std::replace(tz_name.begin(), tz_name.end(), '/', '\\');
#endif
    return get_tzdb().locate_zone(tz_name);
```